packages/svelte/README.md

```diff
@@ -10,67 +10,38 @@ Carbon Charts Svelte is a thin Svelte wrapper around the vanilla JavScript `@car
 
 These Svelte wrappers were developed by Eric Liu.
 
-Please direct all questions regarding support, bug fixes, and feature requests to [@metonym](https://github.com/metonym).
+Please direct all questions regarding support, bug fixes and feature requests to [@metonym](https://github.com/metonym).
 
 ## Getting started
 
 Run the following command using [npm](https://www.npmjs.com/):
 
 ```bash
-npm install -D @carbon/charts-svelte @carbon/styles d3 d3-cloud d3-sankey
+npm install -D @carbon/charts-svelte @carbon/charts @carbon/styles d3 d3-cloud d3-sankey
 ```
 
 If you prefer [Yarn](https://yarnpkg.com/en/), use the following command instead:
 
 ```bash
-yarn add -D @carbon/charts-svelte @carbon/styles d3 d3-cloud d3-sankey
+yarn add -D @carbon/charts-svelte @carbon/charts @carbon/styles d3 d3-cloud d3-sankey
 ```
 
-## Setup
-
-This is an overview of using Carbon Charts with common Svelte setups.
-
-- [SvelteKit](#sveltekit)
-- [Vite](#vite)
-
 ### SvelteKit
 
-[SvelteKit](https://github.com/sveltejs/kit) is the official framework for building apps that
-support client-side rendering (CSR) and server-side rendering (SSR). SvelteKit is powered by
-[Vite](https://github.com/vitest-dev/vitest).
+While this component library can be used with any build environments for Svelte, [SvelteKit](https://kit.svelte.dev) is the official framework for building Svelte apps supporting client-side and server-side rendering (SSR). SvelteKit is powered by
+[Vite](https://vitejs.dev).
 
-Modules `@carbon/charts` and `carbon-components` should not be externalized for SSR when building for production.
+The module `@carbon/charts` should not be externalized for SSR when building for production.
 
 ```js
-// vite.config.js
+// vite.config.mjs
 import { sveltekit } from '@sveltejs/kit/vite'
 
 /** @type {import('vite').UserConfig} */
 const config = {
 	plugins: [sveltekit()],
 	ssr: {
-		noExternal: process.env.NODE_ENV === 'production' ? ['@carbon/charts', 'carbon-components'] : []
-	}
-}
-
-export default config
-```
-
-### Vite
-
-You can also use Vite without SvelteKit. Instruct `vite` to optimize `@carbon/charts` in
-`vite.config.js`.
-
-```js
-// vite.config.js
-import { svelte } from '@sveltejs/vite-plugin-svelte'
-
-/** @type {import('vite').UserConfig} */
-const config = {
-	plugins: [svelte()],
-	optimizeDeps: {
-		include: ['@carbon/charts', 'carbon-components'],
-		exclude: ['@carbon/telemetry']
+		noExternal: process.env.NODE_ENV === 'production' ? ['@carbon/charts'] : []
```

Comment:
I just tried this with my `sveltekit` example in [this repo](https://github.com/metonym/carbon-charts-svelte-examples/tree/master/sveltekit).

Development works fine, but I believe `carbon-components` still needs to be externalized.

After building for production and running `yarn preview`, I saw this error:

```sh
import { Modal as Modal$2 } from "carbon-components";
         ^^^^^
SyntaxError: Named export 'Modal' not found. The requested module 'carbon-components' is a CommonJS module, which may not support all module.exports as named exports.
CommonJS modules can always be imported via the default export, for example using:

import pkg from 'carbon-components';
const { Modal: Modal$2 } = pkg;

```

Reply:
Hi @metonym. I just did the following...
```bash
git clone https://github.com/metonym/carbon-charts-svelte-examples.git
cd carbon-charts-svelte-examples/sveltekit
yarn install
yarn build
yarn preview
```
Then I invoked a modal on one of the charts...

![image](https://github.com/carbon-design-system/carbon-charts/assets/5770938/53194067-a875-404c-8299-af8eaacbda08)


I did not get the error you got related to `carbon-components`. Can you share some of your environmental details? I noticed an older version of `@carbon/charts-svelte` (^1.6.3) in the package.json. Can you check to see if your node_modules has version 1.11.8 of both `@carbon/charts` and `@carbon/charts-svelte`? The core `@carbon/charts` package switched to using vite/esbuild in 1.9.0 and above. That's when the optimizeDeps were added for `carbon-components` there. Wondering if your local env might not be up to date.